.github/workflows/test.yml

```diff
@@ -65,7 +65,7 @@ jobs:
           tar -C ~ --strip-components=1 --wildcards -xf ./shellcheck-*.tar.xz 'shellcheck-*/shellcheck'
           curl -o ~/shfmt -L "https://github.com/mvdan/sh/releases/download/v$SHFMT/shfmt_v${SHFMT}_linux_amd64"
           chmod 755 ~/shfmt ~/shellcheck
-          pip3 install --user compiledb
+          pip3 install --user --upgrade compiledb black click
```

Comment:
Why add `click`? It doesn't seem to be used by anything.

Reply:
Broken VM image, black conflicts with the version of click that's used there. I'll add a comment.

https://github.com/hg/tinc/runs/5806563425?check_suite_focus=true#step:5:4